test/multi_node_test.py

```diff
@@ -25,80 +25,6 @@ def tearDown(self):
     # Kill the Ray cluster.
     subprocess.Popen(["ray", "stop"]).wait()
 
-  def testErrorIsolation(self):
```

Comment:
Let's not remove these tests. It should be enough to change `ray.error_info` to `ray.global_state.error_info()`, right?